index.js

```diff
@@ -840,6 +842,15 @@ const ContainerService = {
     // all the configuration must go away now.
     this._windowMap = {};
 
+    // Remove all cookies with origin attributes
+    ContainerService.queryIdentities().then(identities => {
+      for (const identity of identities) {
+        CookieManager.removeCookiesWithOriginAttributes(JSON.stringify({userContextId: identity.userContextId}));
```

Comment:
The issue here is that we are not removing the identities at all. When an identity is deleted, the cookies are removed.
What we should do here is to remove all the identities or 0, or some of them based on the version of FF and the previous configuration of containers (before the installation of the addon). See #178 .